docs/src/howto.md

```diff
@@ -121,3 +121,96 @@ Action(1:100, (args...)->rect(pos(:my_circle)+Point(-10, -100), 20, 20, :fill))
 
 In this animation the position of the circle is saved inside `:my_circle` and can be used with `pos(:my_circle)` inside the `rect` function.
 
+## How can I show a text being drawn?
+
+A `text` can appear as every other object with `appear(:fade)` and `appear(:scale)` but it also has a special [`appear`](@ref) functionality called 
+`:draw_text`.
+
+You can use 
+```julia
+Action(
+    1:100,
+    (args...) -> text("Hello World!"; halign = :center);
+    subactions = [
+        SubAction(1:15, sineio(), appear(:draw_text)),
+        SubAction(76:100, sineio(), disappear(:draw_text)),
+    ]
+)
+```
+
+to let the text `"Hello World!"` appear from left to right in an animated way. 
+
+## How can I let an object move along a path?
```

Comment:
"How can I let an object move along a path?" -> "How Can I Have an Object Follow a Path?"

Reply:
in how to all headers are like "How can I let an object move along a path?" atm

Reply:
Oh gotcha - it's just that using the word "let" here sounds awkward in English. Perhaps "How can I have an object follow a path?"